sql/core/src/test/resources/sql-tests/results/date_part.sql.out

```diff
@@ -857,7 +857,7 @@ select date_part('epoch', c) from t2
 -- !query 106 schema
 struct<date_part('epoch', t2.`c`):decimal(18,6)>
 -- !query 106 output
-31873892788.332003
+31897220765.004003
```

Comment:
I have compared to PostgreSQL output:
```sql
maxim=# select date_part('epoch', interval '1010 year 9 month 8 day 7 hour 6 minute 5 second 4 millisecond 3 microseconds');
    date_part
-----------------
 31897220765.004
(1 row)
```
It seems PostgreSQL just looses the precision.